docs/concepts/copilot/enable-copilot.md

```diff
@@ -12,32 +12,40 @@ tags:
 
 From the kubernetes cluster where Prophecy services are running:
 
-1. The `prophecy-operator` service must be running version `3.1.0.0` or greater.
-2. The `athena` Prophecy service should be running version `3.1.0.0` or greater. This is only required if using the Pipeline approach to the upgrade.
-3. In the `ProphecyCluster` CR, do the following:
+1. The `prophecy-operator`, `edweb`, `metagraph`, and other Prophecy services must be running version `3.1.0.0` or greater.
+2. The above services should all be running the same release-version as the `copilot` service defined in the image below, eg `3.1.0.0`
+3. The `athena` Prophecy service should be running version `3.1.0.0` or greater. This is only required if using the Pipeline approach to the upgrade.
+4. In the `ProphecyCluster` CR, do the following:
 
 - add `copilot` to `enabledapps`
 - add `image` section if required as below, else defaults to `latest`
 - add the `copilot` section as below in the same level as other enabled apps
 
 ```
 copilot:
-    image: 133450206866.dkr.ecr.us-west-1.amazonaws.com/copilot:3.1.0.0
+    image:  gcr.io/prophecy-share/copilot:<release-version>
 ```
 
-4. Once deployed, wait for the `copilot-cm` to be created. Then edit the default values in the configmap to match the ENVs required.
-5. Supply the API keys for the appropriate ENV:
+5. Once deployed, wait for the `copilot-cm` to be created. Then edit the default values in the configmap to match the ENVs required.
+6. Supply the API keys for the appropriate ENV.
+
+OpenAI
 
 ```
-AZURE_DEPLOYMENT_NAME: "< add value here >"
-AZURE_OPENAI_API_KEY: "< add value here >"
-AZURE_OPENAI_ENDPOINT: ""< add value here >"
-AZURE_OPENAI_VERSION: "< add value here >"
 NUM_WORKERS: "< add value here >"
 OPENAI_API_KEY: "< add value here >"
```

Comment:
Optional model can also be provided. https://simpledatalabs.slack.com/archives/C03RGJ6SQMS/p1692763278318789?thread_ts=1692249186.200939&channel=C03RGJ6SQMS&message_ts=1692763278.318789

Reply:
Thanks for catching that - Fixed